src/main/java/org/mqttbee/mqtt/MqttClientData.java

```diff
@@ -194,4 +226,16 @@ public void to(@NotNull final Channel channel) {
         channel.attr(KEY).set(this);
     }
 
+    private KeyStore getKeyStore() {
```

Comment:
These methods can be removed if SslData and SslConfig are refactored to SslConfig and SslConfigBuilder

Reply:
yes, for later